src/app/pages/ArticlePage/ElectionBanner/config.ts

```diff
@@ -8,7 +8,6 @@ export default {
     'include/vjafwest/1365-2024-us-presidential-election-banner/{service}/app',
   iframeDevSrc:
     'include/vjafwest/1365-2024-us-presidential-election-banner/develop/{service}/app',
-  thingIds: [
-    '647d5613-e0e2-4ef5-b0ce-b491de38bdbd', // https://www.bbc.co.uk/things/647d5613-e0e2-4ef5-b0ce-b491de38bdbd
-  ],
+  thingId: '647d5613-e0e2-4ef5-b0ce-b491de38bdbd',
```

Comment:
While we're in here, would it be too much to rename this to be something like `usElectionThingId`